File: sdkwork-backend-sdk-go/types/pool_config.go
package types


type PoolConfig struct {
	MinIdle int `json:"minIdle"`
	MaxPoolSize int `json:"maxPoolSize"`
	ConnectionTimeout int `json:"connectionTimeout"`
	IdleTimeout int `json:"idleTimeout"`
	MaxLifetime int `json:"maxLifetime"`
	AutoCommit bool `json:"autoCommit"`
	ConnectionTestQuery string `json:"connectionTestQuery"`
	PoolName string `json:"poolName"`
}

File: sdkwork-backend-sdk-csharp/Models/PoolConfig.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PoolConfig
    {
        public int? MinIdle { get; set; }
        public int? MaxPoolSize { get; set; }
        public int? ConnectionTimeout { get; set; }
        public int? IdleTimeout { get; set; }
        public int? MaxLifetime { get; set; }
        public bool? AutoCommit { get; set; }
        public string? ConnectionTestQuery { get; set; }
        public string? PoolName { get; set; }
    }
}
